lib/route.js

```diff
@@ -231,6 +234,16 @@ function buildRouting (options) {
         return
       }
 
+      const { exposeHeadRoute } = opts
+      const hasRouteExposeHeadRouteFlag = exposeHeadRoute != null
+      const shouldExposeHead = hasRouteExposeHeadRouteFlag ? exposeHeadRoute : globalExposeHeadRoutes
+      const headRouteExists = router.find('HEAD', path) != null
```

Comment:
Just a thought: if there is a HEAD route and the user sets the route level option `hasRouteExposeHeadRouteFlag` too, would it be too harsh logging a warning?

Reply:
Don't think so, would be just adding an `else if` to see if the `headRouteExists` and `hasRouteExposedHeadRouteFlag` it's set to `true` for printing the warning. What do you think?

Reply:
I think it could let users say "oh gosh, it saves me a bad headache"
If you had the chance to add it, I think it will be really appreciated by someone!

Reply:
haha, sure! No issue at all, I think will save someone's day, let me add it 😄

Reply:
Done in 976b080
Please have a look 🙂